### DISCUSSION TITLE
I think our mvp is about ready

### DISCUSSION THREAD

**Discussion by Riley23062** [csmith1188/Formbar.js#59] [General]
I feel like our MVP is on the way to being finished, is there anything else you guys think we need to add besides the lightbar functionality????

It seems to me like the actual website version of this is complete and we should shift our focus

**Comment by Eshimmel** [comment]
Sort by
colors/sfx/bgm
Poll history
Quiz API
Quiz Loader / Lesson loader
API rate limit
